packages/core/src/components/Menu/MenuItem/__tests__/MenuItem.test.js

```diff
@@ -24,6 +24,49 @@ describe("<MenuItem />", () => {
     expect(queryAllByText("Sub Item")).toHaveLength(0);
   });
 
+  const submenuPositions = [
+    {
+      submenuPosition: "left",
+      expectedPosition: "left-start"
+    },
+    {
+      submenuPosition: undefined,
```

Comment:
is it still like that?

Reply:
yes, it is checking the case where the user doesnt send the prop